File: client_curve_fever/src/pl/poznan/put/game/PointWrapper.java
package pl.poznan.put.game;

public class PointWrapper {
    private int number;
    private Point point;

    public PointWrapper(int number, Point point) {
        this.number = number;
        this.point = point;
    }

    public int getNumber() {
        return number;
    }

    public void setNumber(int number) {
        this.number = number;
    }

    public Point getPoint() {
        return point;
    }

    public void setPoint(Point point) {
        this.point = point;
    }
}

File: server_curve_fever/PointWrapper.cpp
#include "PointWrapper.h"

PointWrapper::PointWrapper(int playerNumber, int pointNumber, Point point) : point(point) {
    this->playerNumber = playerNumber;
    this->pointNumber = pointNumber;
    this->point = point;
}

int PointWrapper::getPlayerNumber() {
    return playerNumber;
}

int PointWrapper::getPointNumber() {
    return pointNumber;
}

Point PointWrapper::getPoint() {
    return point;
}